ui/pages/Search/components/filters/LocusListSelector.jsx

```diff
@@ -24,8 +23,34 @@ class BaseLocusListDropdown extends React.Component {
 
   componentDidUpdate(prevProps) {
     const { locusList, onChange } = this.props
+
     if (prevProps.locusList.rawItems !== locusList.rawItems) {
-      const { locusListGuid, rawItems } = locusList
+      const { locusListGuid } = locusList
+      let { rawItems } = locusList
+
+      if (locusList.paLocusList) {
+        const grouped = locusList.items?.reduce((acc, item) => {
```

Comment:
I think it would be neater to just reduce once into the object you want: 
```
const CONFIDENCE_COLORS = { '1': 'red', '2': 'amber', '3': 'green', '4': 'green'}
rawItems = locusList.items?.reduce((acc, item) => {
  const color = CONFIDENCE_COLORS[item.pagene?.confidenceLevel || 0]
  if (color in acc) {
    acc[color] = [acc[color], item.display].filter(val => val).join(', ')
  }
  return acc
}, { red: '', amber: '', green: ''})
```